src/Services/UnixTimestampNormalizer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Meilisearch\Bundle\Services;
+
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class UnixTimestampNormalizer implements NormalizerInterface
+{
+    /**
+     * @param \DateTimeInterface $object
+     */
+    public function normalize(mixed $object, string $format = null, array $context = []): int
+    {
+        return (int) $object->format('U');
```

Comment:
```suggestion
        return $object->format('U')->getTimestamp();
```
?

Reply:
`$object->getTimestamp()` precisely :-) (didn't even know that method existed!)

Reply:
yeah, made a typo :D